### PR TITLE
Restore "Search by repo title" functionality

### PR DESCRIPTION
**Pull request by joeuhren**
@azzy9 the "Search by repo title" option is now working again. This may be an oversimplification of the original functionality because I'm a little unsure of how the original plugin worked. The new functionality basically finds all repos that match the name given and then returns all zip files from the most recent release for each repo. Assuming it returns 1 or more zip file matches, you can click the zip file name to install it, which it does. There are 2 main problems or shortcomings with this plugin installation though:

1) Kodi must be closed and reopened for the newly installed plugin to show up in the list of plugins/add-ons. This doesn't seem to be a new problem since the original code prompts you to restart or continue if you want to install more plugins. I did bit of searching and so far I haven't been able to find a method of refreshing the plugin list while Kodi is running. I also notice that on windows 10 choosing the Restart option always freezes Kodi and I have to manually end the task via task mgr although perhaps it is just me (?)

2) The plugin is installed in the disabled state, and must be manually enabled after restarting Kodi. The old code has a method of calling the "Addons.SetAddonEnabled" api which DOES NOT WORK with the initial install as it always fails with this msg: `{'error': {'code': -32602, 'message': 'Invalid params.'}, 'id': 1, 'jsonrpc': '2.0'}`. I explored this a bit already and the code to automatically activate the plugin does actually work, but only after restarting Kodi. Therefore, it seems that it is not possible to auto-enable the plugin after installing until Kodi is aware of the plugin in the list of add-ons. One possible workaround idea would be to have the Git Browser "remember" the plugins that it installed and upon restarting Kodi it can review the list of installed plugins and try enabling them at that time. This seems like overkill to me and it would be much nicer if the there was a way to refresh the internal list of plugins without restarting which I'm sure would allow the "Addons.SetAddonEnabled" api to work right afterwards

Long story short, I feel this is a step in the right direction, even if it isn't as perfect as I would like. I'd love to hear if you have any ideas of how to resolve either of the 2 main issues

### CODE REVIEW

**Comment by joeuhren** [comment]
After reviewing the existing code a bit more in depth I realized that there already was code to refresh the plugin list but it just wasn't working with Kodi Nexus which is what I'm using for testing. I found that a number of other built-in cmds were also not working such as notifications because all the xbmc.executebuiltin() function calls were prefixed with XBMC like xbmc.executebuiltin('XBMC.UpdateLocalAddons') and the docs show it as xbmc.executebuiltin('UpdateLocalAddons') without the prefix and it works. I couldn't find information about when that change was made in Kodi which is sort of weird to me. Nearest I can figure is it always should have been without the XBMC prefix for the last 10 years of Kodi versions and might have been a deprecated way of calling built-in functions way back from the XBMC days before Kodi was renamed (?). Regardless, with the newest commit the "Search by repo title" option now works 100% without any shortcomings that I can see.

**Comment by azzy9** [comment]
Hi, by looking at the thread: [https://forum.kodi.tv/showthread.php?tid=328392](https://forum.kodi.tv/showthread.php?tid=328392) it looks like you are correct

**Comment by azzy9** [comment]
Thanks again for the PR, works awesome

**Comment by azzy9** [comment]
Just thinking @joeuhren, would you like me to add you as a collaborator to the project?

**Comment by joeuhren** [comment]
> Just thinking @joeuhren, would you like me to add you as a collaborator to the project?

That would be great, although I cannot promise I will have tons of time to work on it all the time but I can chip away at it as time permits.

**Comment by azzy9** [comment]
@joeuhren, invitation sent :)
